### PR TITLE
Add manager cli for configuring Stratagem

### LINKED ISSUE #1060: Add Cli commands to enable / disable stratagem, and configure the systemd timer
Add commands to the IML manager CLI to enable / disable stratagem, and to configure the systemd timer.

### LINKED ISSUE #1079: chroma_core_stratagemconfiguration interval should be bigint
The `chroma_core_stratagemconfiguration` `interval` field should be a `bigint` for consistency with the rest of the *_duration fields.

### LINKED ISSUE #1080: Unable to delete rows from chroma_core_stratagemconfiguration
`chroma_core_stratagemconfiguration` rows cannot be deleted due to a foreign key reference in `chroma_core_configurestratagemjob`.

We should probably add the `not_deleted` functionality to the `StratagemConfiguration` model so it can be soft deleted.

### PR DESCRIPTION
**Pull request by jgrund**
Fixes #1079.
  Fixes #1060.
  Fixes #1080.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: chroma_core/south_migrations/0048_auto__add_field_stratagemconfiguration_not_deleted.py [hunk 1]
```diff
@@ -0,0 +1,1515 @@
...
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'StratagemConfiguration.not_deleted'
+        db.add_column(u'chroma_core_stratagemconfiguration', 'not_deleted',
+                      self.gf('django.db.models.fields.NullBooleanField')(default=True, null=True, blank=True),
```
Comment (johnsonw) [inline chroma_core/south_migrations/0048_auto__add_field_stratagemconfiguration_not_deleted.py]: could this be in 0047?

Comment (jgrund) [reply chroma_core/south_migrations/0048_auto__add_field_stratagemconfiguration_not_deleted.py]: Should be in 0046 if anything. I'll take a look

Comment (johnsonw) [reply chroma_core/south_migrations/0048_auto__add_field_stratagemconfiguration_not_deleted.py]: ok awesome.

**Comment by jgrund** [comment]
stratagem-config-cli

File: iml-manager-cli/src/stratagem.rs [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+                    serde_json::json!({ "limit": 0 }),
+                );
+                let result: Result<ApiList<StratagemConfiguration>, _> = run_cmd(fut);
+
+                stop_spinner(None);
+
+                match result {
+                    Ok(r) => {
+                        if r.objects.is_empty() {
+                            return println!("0 Stratagem configuration records found");
```
Comment (johnsonw) [inline iml-manager-cli/src/stratagem.rs]: Any reason why we don't call `exit(exitcode::OK)`?

Comment (jgrund) [reply iml-manager-cli/src/stratagem.rs]: No difference really:

```
[root@adm chroma-manager]# iml stratagem configure list
0 Stratagem configuration records found
[root@adm chroma-manager]# echo $?
0
```

**Pull request merged**